tests/scripts/check_stats.py

```diff
@@ -1,6 +1,6 @@
 #! /usr/bin/env python
```

Comment:
On Ubuntu 19.10, there is no more python 2.7 installation by default installation. Do you think we should update all the scripts to force python3. Nexus has done this. On systems with both python 2.7 and python3 installed. python is pointing python 2.7.

Reply:
Official guidance here: https://www.python.org/dev/peps/pep-0394/

I would prefer to leave it as 'python' because that is the most generic, and these scripts still work with python 2.   However, runtime distributions have the option of not providing a generic 'python' command at all, which then doesn't work.

I think we should leave it as 'python' for now, and see how many people run into problems, and how many environments do not provide a generic 'python'.

Reply:
We'll make python3 a hard requirement (many reasons). This PR need not change.

Reply:
Reconsidering: it will save some time to update the shebang here. I'll make the one character edit.